### PR TITLE
Octopus: modify planning component to support multi-adc simulation

### PR DESCRIPTION
**Pull request by changsh726**
I'm not sure this is a proper approach to change the output channels of octopus dependent components, and if not, I'd like your advice.

### CODE REVIEW

File: cyber/common/global_data.cc [hunk 1]
```diff
@@ -67,6 +67,8 @@ GlobalData::GlobalData() {
   is_reality_mode_ =
       config_.run_mode_conf().run_mode() == proto::RunMode::MODE_REALITY;
 
+  is_octopus_mode_ = false;
```
Comment (xiaoxq) [inline cyber/common/global_data.cc:70]: Please refer to https://github.com/ApolloAuto/apollo/pull/11131#discussion_r415933639
We'd better avoid such change in Cyber, unless approved by @unacao and @vlin17.

File: cyber/common/global_data.h [hunk 2]
```diff
@@ -58,6 +58,12 @@ class GlobalData {
   void EnableSimulationMode();
   void DisableSimulationMode();
 
+  void EnableOctopusMode();
+  bool IsOctopusMode() const;
+
+  void SetADCId(const int& adc_id);
+  int ADCId() const;
```
Comment (unacao) [inline cyber/common/global_data.h:65]: cyber global data is a singleton, if we set the adc_id here, it cannot be set to different values for different planning modules.

File: cyber/common/global_data.cc (hunk 1 above)
Comment (jmtao) [reply cyber/common/global_data.cc:70]: better not use code name/project name as variable name here. 
it's hard to understand what "octopus" is.  poor readability

**Comment by unacao** [comment]
According to @ycool, you can add the adc_id field in https://github.com/ApolloAuto/apollo/blob/master/modules/planning/proto/planning_config.proto so that config for each ADC can be different.

File: modules/planning/planning_component.cc [hunk 3]
```diff
@@ -56,45 +57,89 @@ bool PlanningComponent::Init() {
...
-      });
-
-  pad_msg_reader_ = node_->CreateReader<PadMessage>(
-      FLAGS_planning_pad_topic,
-      [this](const std::shared_ptr<PadMessage>& pad_msg) {
-        ADEBUG << "Received pad data: run pad callback.";
-        std::lock_guard<std::mutex> lock(mutex_);
-        pad_msg_.CopyFrom(*pad_msg);
-      });
+  if (GlobalData::Instance()->IsOctopusMode()) {
```
Comment (jmtao) [inline modules/planning/planning_component.cc:60]: non OctopusMode, these readers shall be created as well.

**Pull request closed without merging**